repair/repair.cc

```diff
@@ -8,6 +8,7 @@
 
 #include "repair.hh"
 #include "repair/row_level.hh"
+#include "repair_task.hh"
```

Comment:
nit: `repair/` path prefix

Reply:
Also, regarding patch subject, the user requested task is derived from the root task, which is only internal to the repair tasks implementation, so I don't think it "deserves" to be mentioned in the subject and commit log.

It would be clearer to describe this as e.g. `repair: create task for user requested repair` or `repair: add user_requested_repair_task_impl`